pom.xml

```diff
@@ -1021,6 +1021,25 @@
 			<artifactId>maven-gpg-plugin</artifactId>
 			<version>1.6</version>
 		</dependency>
+
+		<!-- https://github.com/protocolbuffers/protobuf/tree/master/java -->
+		<dependency>
+			<groupId>com.google.protobuf</groupId>
+			<artifactId>protobuf-java</artifactId>
+			<version>3.11.0</version>
```

Comment:
It is not 100% updated on their GitHub.
cold you try the 3.12.X out?
<https://mvnrepository.com/artifact/com.google.protobuf/protobuf-java>